src/E2_economics.py

```diff
@@ -57,138 +64,401 @@
 """
 
 
+class UnexpectedValueError(UserWarning):
```

Comment:
If this is an "Error" it should inherit from the `ValueError` class

Reply:
Changed to TypeError in #525, 44465abb6c1382b61a2cc87a897fbb7de8f1708d